app/jobs/notifications/moderation_notification_job.rb

```diff
@@ -0,0 +1,22 @@
+module Notifications
+  class ModerationNotificationJob < ApplicationJob
+    queue_as :send_moderation_notification
+
+    def perform(notifiable_id, service = Notifications::Moderation::Send)
+      available_moderators = Notifications.available_moderators
```

Comment:
I'd move `available_moderators` this inside the module Notifications::Moderation if you want to have it as a module method